Nos/Views/Components/Form/NosTextEditor.swift

```diff
@@ -11,6 +11,7 @@ struct NosTextEditor: View {
             TextEditor(text: $text)
                 .textInputAutocapitalization(.none)
                 .foregroundColor(.primaryTxt)
+                .scrollContentBackground(.hidden)
```

Comment:
I noticed this bug a few days ago but hadn't filed a ticket yet. I decided to slip the fix in here. 🙈

Reply:
You can see the bug-  a black background for the bio field - in the `before` screenshot in the PR description.

Reply:
Oh I saw this bug, too, so I'm glad this is being fixed.